src/Containers/Microsoft.NET.Build.Containers/DigestUtils.cs

```diff
@@ -17,6 +17,16 @@ internal sealed class DigestUtils
     /// </summary>
     internal static string GetDigestFromSha(string sha) => $"sha256:{sha}";
 
+    internal static string GetShaFromDigest(string digest)
+    {
+        if (!digest.StartsWith("sha256:", StringComparison.OrdinalIgnoreCase))
+        {
+            throw new ArgumentException("Invalid digest format. Digest must start with 'sha256:'.");
```

Comment:
I saw localized exception messages in this project, so I think this should be localized too.

Reply:
thanks for the suggestion! 
I think this is fine not localized, as the exception is not intended for the customers. this is just a check for an engineer if they misuse it.
If you or anyone else feels strongly about this please unresolve the conversation.